cpp/src/io/parquet/writer_impl.cu

```diff
@@ -171,7 +171,7 @@ struct aggregate_writer_metadata {
     std::vector<std::vector<uint8_t>> column_indexes;
   };
   std::vector<per_file_metadata> files;
-  std::string created_by                                   = "";
+  std::string created_by = "libcudf version " CUDF_STRINGIFY(CUDF_VERSION);
```

Comment:
There is also a similar field in `aggregate_writer_metadata`.

Reply:
This is `aggregate_writer_metadata` 😅

Reply:
Oh sorry didn't realize that :smile: . Then, how about the same field in `FileMetaData` in `parquet.hpp`?

Reply:
That one has its `created_by` populated in `aggregate_metadata::get_metadata()` on the write path. `FileMetaData` is also used by the reader, so I don't think we need to provide a default value. Later we should probably make the field in `FileMetaData` optional.

Reply:
As mentioned, the field in `FileMetaData` is also used by the reader but how about this? If it is used only for writer, and if we initialize this field like this, should we make it `const`?

Reply:
I think the one here can be. I can't imagine wanting to allow users to change it.

Reply:
Why make it `const`? It only cripples the efficiency of `aggregate_writer_metadata`s moves. I don't see what we're protecting from, it's a private data member.

Reply:
Does it even need to be a member?  All it's used for is to initialize the `FileMetaData` object. Could just use the literal in `get_metadata()` and be done.